### PR TITLE
Create table

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
# Database created

## Table created

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/schema.sql#L3-L10

## Adding data

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/data.sql#L3-L6

## Adding queries

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L3
q1

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L4
q2

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L5
q3

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L6
q4

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L7
q5

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L8
q6

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L9
q7

https://github.com/Alejandro-Bernal-M/vetclinic/blob/ebdf5e9f81b1c92bfa9e5d232c639e924d119e35/queries.sql#L10
q8

## Error correction, initially the name "Gabumon"  was misspelled as "Gagumon" and was corrected using 
  ` UPDATE animals SET name = 'Gabumon' WHERE id = 2; `
 
the update query `Find all animals not named Gabumon.`  show this result:
image

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @Alejandro-Bernal-M,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] You forgot to add the license section, Kindly add it with MIT licensing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+	id INT PRIMARY KEY,
```
Comment (alexander16108) [inline schema.sql:4]: [optional] - i suggest you make the id column not null and autoincremented by using `    id int NOT NULL IDENTITY(1,1)
`

**Review by alexander16108 — APPROVED** [review]
Hi @Alejandro-Bernal-M , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Alejandro-Bernal-M) [reply schema.sql:4]: Thank you for the suggestion

**Pull request merged**